python/lsst/decam/hits/__init__.py

```diff
@@ -0,0 +1,7 @@
+from __future__ import absolute_import
+
+#
+# See COPYRIGHT file at the top of the source tree
+#
```

Comment:
IIRC we're not supposed to use this comment (RFC pending lawyer approval).

Reply:
Also no longer relevant.